md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
```

Comment:
Conflict resolution guidelines?  I'm not sure what that entails, but maybe write something about and then we'd get an idea of where it belongs?